cluster.go

```diff
@@ -1738,3 +1743,50 @@ func diffPeers(peers1, peers2 []peer.ID) (added, removed []peer.ID) {
 	}
 	return
 }
+
+func (c *Cluster) multiCallWrapper(
+	ctxs []context.Context,
+	dests []peer.ID,
+	svcName, svcMethod string,
+	args interface{},
+	replies []interface{}) []error {
+
+	if !checkMatchingLengths(len(ctxs), len(dests), len(replies)) {
+		panic("ctxs, dests and replies must match in length")
+	}
+
+	errs := make([]error, len(dests))
+	var actualCtxs []context.Context
+	var actualDests []peer.ID
+	var actualReplies []interface{}
+
+	authErr := fmt.Sprintf("client does not have permissions to this method, service name: %s, method name: %s", svcName, svcMethod)
```

Comment:
make this a global variable at the top. Let it say that it is skipping because it was not authorized before.

Reply:
Like the message below. Avoid returning one error to the user and logging a different one.

Reply:
The message is same as it would have been be returned by server if request were sent. I think this similarity will benefit us while checking authorization errors (will save us from checking an extra separate error message)

> Let it say that it is skipping because it was not authorized before.
We are skipping, is more of a warning than error (since we ourselves are doing it)

Should I still change?

Reply:
Never check if an error is certain type of error by checking the string (if you can). You are making assumptions that break sooner or later.  The library gives you a way to perform the check. `rpc.IsAuthorizationError(...)`. And you are lying to the user, the request did not fail because it was not authorized now, but because you did not perform it based on a previous failure.